ext/strings.go

```diff
@@ -68,12 +68,12 @@ import (
 //
 // LastIndexOf
 //
-// Returns the integer index of the last occurrence of the search string. If the search string is
-// not found the function returns -1.
+// Returns the integer index at the start of the last occurrence of the search string. If the
+// search string is not found the function returns -1.
 //
-// The function also accepts an optional index which represents where to begin the search in
-// through the string. If the substring is the empty string, the index where the search starts
-// is returned (string length or custom).
+// The function also accepts an optional index which represents where to begin the substring
+// search starting from the end of the string. If the substring is the empty string, the index
```

Comment:
I still don't like the sentence.  And there's still some ambiguity about the behavior for overlapping matches.  I *think* what we want is the mirror image of `indexOf()`.

How about this?

The function also accepts an optional index which discards matches starting at or after that index.  The function will return the substring match with the greatest index which is less than the optional limit.

Also add an example/test:  `'bananananana'.lastIndexOf('nana', 8) // returns 6